contracts/GolemNetworkToken2.sol

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Golem Factory
+// Licensed under the GNU General Public License v3. See the LICENSE file.
+
+pragma solidity ^0.4.19;
+
+import "./ExtendedToken.sol";
+import "./TokenProxy.sol";
+
+contract GolemNetworkToken2 is TokenProxy, ExtendedToken {
+
+    string public constant name = "Golem Network Token 2";
```

Comment:
Please don't. Let's keep the previous name or use Golem Network Token Batching, but not this. It's misleading.

Reply:
:)
GolemNetworkTokenNew